conda-forge.yml

```diff
@@ -1,9 +1,9 @@
 travis:
   secure:
-    BINSTAR_TOKEN: cXmMh0qV5bhXevVtdcMf8VOVIG0zOA0cQRihCsOaDG0yPSP1tW0N40llDkDhjk80wIKMMzuwofaarESzW5i5+GTbvDMmzpeHZAEoXYUIsNTxNJMX7KcvxNdeVIGNM8tcM3a4Zp/u0cvOjDs6xSHeZPdNit3x8rDaARASwU+YZnIIY8Iieizo3LmWxRghjOdFYnr0frOshy38PI+l1ywYqsN/L8+4GqxWtkTuzK3fnI3MlwfDC8tVImjXuukHK3J78SsDdnbTBJlV0tcNV8iGa08Md55bgv9HEgFbHRSunc22lLjFRHMT10Hvl0H0HdylYkQTKC7iOwIw9D3kIp4uR+g0rDKzBuuYF+jV65+UhR5TKMiH4QIiMxBMLpB8QU1Ey58L2X2t1j06d+VZYp2UiubGNvxjXun9tWFnIotFoho1bfW0gUURNvk+fEs0O/0LpReTweNVJLc1JakT7eitFm5gaV8CNzAsNcBCpPsHWiCMQm+FD3D57q+DIJ4ONl0OOBYSDNkziJEvgB0XyV7UpGobm0xu/JTPg+Qq2op3GDxgCfILqhApTEVkMKQJv+o5yHO/JIonxCbE41xIotw8JjhQ2VROFLPGh3F3buiJ26VUbI41yoN20D4y0Bvj7HPdiBHVWNcUmkj1xYZckivnxjCE5eth/TBJiK9gqjgRw6w=
+    BINSTAR_TOKEN: xHG5feKFSl87s9IK6ReTanlNhHkXp4xw5+pdNJxXzpUOHhhDznnJj7+929IxeEuoHJToVty+6BpbLOZGC4B/EirIxY7csjf5ymWHrcTo/hfQ5PL3nlTseoL5l2p2pPwFvzrBREt0O5BBCeiM2rMbr2m814d+mx/VSXvFttfrBbrK3Yn2aU4xdyU0Ce2Rap7QMO9PGwvb1quZWfdyD4DGVx3v6qap+L9IPmOUzJ+Iut58EXD04h3yfdKYulcqv7rlAlQJWokqrlKBsKfRlbP1MaYnKPkx2fcsCneH+8ItJCjoHnTRcGVj6kC6PtQP5cQkScWcWLJPzGhggAywgoTqAGTaGkxm/zEMbObXQGhnEgFcQaD5kn6lEkRFlggobVMDPf8C6wanXqh037So0Qyn/IFqDE5/YyhqPfM2/dgeu7VKEqhqqtK/1/pUiicmDMCKLIbCN9v7aYylZMVWi9BDxubP0hyNz/PssCdu6XmSwwUGgsZVr3JqXWPWX4mskwnny1S45BDzdj1dZaRCeWnzhdN1NeooBZWFH6LW9ojD014rD58TejscKYQkZfOlgbAG5lj4fD8urEBdp8ntLySJywN2gCpbFzvBkyBen6ilDgp5A0H3mOEohPEV53q9XdIXbayeLK524B/eLGmN5s2vpGNh4Hdm0CKg0TYRfwxhWU4=
 appveyor:
   secure:
-    BINSTAR_TOKEN: MP4hZYylDyUWEsrt3u3cod2sbFeRwUziH02mvQOdbjsTO/l1yIxDkP/76rSIjcGC
+    BINSTAR_TOKEN: tumuXLL8PU75WMnRDemRy02ruEq2RpNxeK3dz0MjFssnosPm2v4EFjfNB4PTotA1
 channels:
   targets:
-  - [conda-forge, rc]
+    - [conda-forge, rc]
```

Comment:
Hmm...this should be sufficient. 😕

Reply:
indentation problem?

Reply:
removed the spaces, ``conda smithy rerender`` said "no changes". It also said "no changes" when I added the line.

Reply:
For clarity, what you have here seems correct to me. Think `conda-smithy` is ignoring it for some reason, but don't fully understand why. Though there have been some changes with channels in recent `conda-smithy` releases, which could have changed how this gets handled. Let's see if someone else can spot what is going on. 😄

Reply:
is there a way to force a rebuild?